EPRN.PRNS.API/Controllers/PRNController.cs

```diff
@@ -164,6 +164,32 @@ public async Task<ActionResult> SaveDecemberWaste(int? id, bool decemberWaste)
             return Ok();
         }
 
+        [HttpGet]
+        [Route("Reference")]
+        public async Task<IActionResult> GetPrnReference(int id)
+        {
+            var prnReference = await _prnService.GetPrnDetails(id);
+            
+            if (prnReference == null)
+                return NotFound();
+
+            return Ok(prnReference.ReferenceNumber);
+        }
+
+        [HttpPost]
+        [Route("SentTo/{status}")]
+        public async Task<ActionResult> SaveSentTo(int? id, PrnStatus? status)
```

Comment:
Don't make the fields nullable on the API unless that is the actual intension. I suspect not though

Reply:
Done.